browserify build broken

From `0.1.7` to `0.2.1` I can't use animation-frame with browserify anymore. Tried it on a clean Ubuntu instance with node 0.12.0.

```
$ npm install animation-frame
$ echo "require('animation-frame')" > index.js

$ browserify index.js 

Error: Cannot find module './native' from '/tmp/node_modules/animation-frame'
    at /usr/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:43:25
    at load (/usr/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:99:43)
    at /usr/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:105:22
    at /usr/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at Object.oncomplete (fs.js:108:15)
```

It does work though if I require it with `require('./node_module/animation-frame/index.js')` but not with `require('./node_module/animation-frame')`.

Oh well, I think I just found the problem. PR incoming.

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "requestAnimationFrame",
     "cancelAnimationFrame"
   ],
-  "main": "AnimationFrame.js",
+  "main": "index.js",
   "licenses": [
     {
       "type": "MIT",

